services/121-service/.eslintrc.js

```diff
@@ -60,6 +61,13 @@ module.exports = {
         // 'promise/prefer-await-to-then': 'warn', // TODO: Enable (locally only) to see if there is something to refactor.
         'promise/valid-params': 'error',
         'prettier/prettier': ['error', { endOfLine: 'auto' }],
+        'no-relative-import-paths/no-relative-import-paths': [
+          'warn',
+          {
+            prefix: '@121-service',
```

Comment:
I set @121-service as the root folder instead of .`/src`. The reason is that this path mapping should also work for the API tests defined in the test folder:

- I did some trial and error with 2 prefixes: @121-service-test and @121-service. 
- However this `'no-relative-import-paths/no-relative-import-paths'` setting only allows for one rootDir so I decide it made more sense to set @121-service to the root instead of having 2 prefixes

@aberonni